accessibility/accessibility-checklist.md

```diff
@@ -39,6 +39,30 @@ Resources:
 * [Tool: HTML5 accessibility feature support for sections](http://www.html5accessibility.com/#sections)
 
 
+### Provide a Skip Navigation option
+
+Not all users are able to navigate by landmarks. For those users the multiple links typically found at the head of a document all have to be read via assistive technology, or laboriously tabbed through using a keyboard.
```

Comment:
```suggestion
Not all users are able to easily navigate by landmarks. Multiple navigation links typically found at the top of a document all have to be laboriously tabbed through using a keyboard, on every page the user visits.
```

Removed the bit about assistive tech just coz everything needs to be read by AT anyway. That changes the focus to how much tedious work it is to physically tab through everything. Also added a bit about how it needs to be done over and over on every page to make it even more 😩